dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/DimensionalObjectUtils.java

```diff
@@ -79,6 +80,13 @@
     private static final Pattern COMPOSITE_DIM_OBJECT_PATTERN = Pattern
         .compile( "(?<id1>\\w+)\\.(?<id2>\\w+|\\*)(\\.(?<id3>\\w+|\\*))?" );
 
+    private static final ImmutableSet<QueryOperator> IGNORE_OPERATOR_SET = ImmutableSet.<QueryOperator> builder()
```

Comment:
Can we switch to just using https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Set.html#of(E...)
I prefer to use language/standard library provided methods if they exist and do what I need.

I would also rename it to something like `SUPPORTED_OPERATORS` as thats what they are. IThat reads nicer below then: if the `operator` passed in below is not supported we ignore it.